locales/en-US/formautofill.properties

```diff
@@ -2,32 +2,116 @@
+# when an address change is detected.
 updateAddressMessage = Would you like to update your address with this new information?
 createAddressLabel = Create New Address
 updateAddressLabel = Update Address
+# LOCALIZATION NOTE (saveCreditCardMessage, saveCreditCardLabel, cancelCreditCardLabel, neverSaveCreditCardLabel):
+# Used on the doorhanger when users submit payment with credit card.
+# LOCALIZATION NOTE (saveCreditCardMessage): %S is brandShortName.
+saveCreditCardMessage = Would you like %S to save this credit card? (Security code will not be saved)
+saveCreditCardLabel = Save Credit Card
+cancelCreditCardLabel = Don’t Save
+neverSaveCreditCardLabel = Never Save Credit Cards
+# LOCALIZATION NOTE (openAutofillMessagePanel): Tooltip label for Form Autofill doorhanger icon on address bar.
 openAutofillMessagePanel = Open Form Autofill message panel
+
+# LOCALIZATION NOTE (autocompleteFooterOption, autocompleteFooterOptionOSX): Button at the bottom of the
+# drop down suggestion. Leads users to Form Autofill browser preferences.
 autocompleteFooterOption = Form Autofill Options
-autocompleteFooterOptionShort = More Options
 autocompleteFooterOptionOSX = Form Autofill Preferences
+# LOCALIZATION NOTE (autocompleteFooterOptionShort, autocompleteFooterOptionOSXShort): Narrow version of the
+# button at the bottom of the drop down suggestion. Leads users to Form Autofill browser preferences.
+autocompleteFooterOptionShort = More Options
 autocompleteFooterOptionOSXShort = Preferences
-address = address
-name = name
-organization = company
-tel = phone
-email = email
+# LOCALIZATION NOTE (category.address, category.name, category.organization, category.tel, category.email):
+# Used in autofill drop down suggestion to indicate what other categories Form Autofill will attempt to fill.
+category.address = address
+category.name = name
+category.organization = company
+category.tel = phone
+category.email = email
 # LOCALIZATION NOTE (fieldNameSeparator): This is used as a separator between categories.
 fieldNameSeparator = ,\u0020
 # LOCALIZATION NOTE (phishingWarningMessage, phishingWarningMessage2): The warning
 # text that is displayed for informing users what categories are about to be filled.
 # "%S" will be replaced with a list generated from the pre-defined categories.
-# The text would be e.g. Also fill company, phone, email
+# The text would be e.g. Also autofills company, phone, email.
 phishingWarningMessage = Also autofills %S
 phishingWarningMessage2 = Autofills %S
+# LOCALIZATION NOTE (insecureFieldWarningDescription): %S is brandShortName. This string is used in drop down
+# suggestion when users try to autofill credit card on an insecure website (without https).
+insecureFieldWarningDescription = %S has detected an insecure site. Form Autofill is temporarily disabled
+
+# LOCALIZATION NOTE: These strings are used in browser preferences.
```

Comment:
Could localization note be used this way without attaching it to a particular string ID?

Reply:
Tools are still relatively stupid, so they associate the comment to the following string. Ideally, you should specify the ID in the comment. See https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practices#Add_localization_notes